docs/operator-manual/argocd-cm.yaml

```diff
@@ -221,6 +221,13 @@ data:
   # An optional comma-separated list of metadata.labels to observe in the UI.
   resource.customLabels: tier
 
+  # An optional comma-separated list of metadata.labels keys to add onto k8s events generated for Applications.
+  # The keys are compared against Application and it's AppProject, if matched,
+  # the corresponding labels are added on the generated event. 
+  # In case of conflict between labels on Application and AppProject, 
+  # the Application label values are prioritized and added to the event.
+  resource.eventLabelKeys: team,env
```

Comment:
Wdyt about `resource.includeLabelKeys` and `resource.excludeLabelKeys` could have wildcards as an improvement - @leoluz

Reply:
Added new keys `resource.includeEventLabelKeys` & `resource.excludeEventLabelKeys`.